pkg/plugins/optional/helm/v1alpha/plugin.go

```diff
@@ -54,9 +52,6 @@ func (Plugin) Version() plugin.Version { return pluginVersion }
 // SupportedProjectVersions returns an array with all project versions supported by the plugin
 func (Plugin) SupportedProjectVersions() []config.Version { return supportedProjectVersions }
 
-// GetInitSubcommand will return the subcommand which is responsible for initializing and scaffolding helm manifests
-func (p Plugin) GetInitSubcommand() plugin.InitSubcommand { return &p.initSubcommand }
-
```

Comment:
Since we have only edit we might want to rename: https://github.com/kubernetes-sigs/kubebuilder/blob/4e369844ee07ab4fff8b3d34d6cedd01d4ccc0af/pkg/plugins/optional/helm/v1alpha/scaffolds/init.go

Too .. we no longer need init,go 
Just edit :-)